src/nvim/main.c

```diff
@@ -1806,54 +1806,100 @@ static void usage(void)
+        "Improved Ex mode\n"));
+  mch_msg(_("  -s                    "
+        "Silent (batch) mode (only for ex mode)\n"));
+  mch_msg(_("  -d                    "
+        "Diff mode\n"));
+  mch_msg(_("  -R                    "
+        "Read-only mode\n"));
+  mch_msg(_("  -Z                    "
+        "Restricted mode\n"));
+  mch_msg(_("  -m                    "
+        "Modifications (writing files) not allowed\n"));
+  mch_msg(_("  -M                    "
+        "Modifications in text not allowed\n"));
+  mch_msg(_("  -b                    "
+        "Binary mode\n"));
+  mch_msg(_("  -l                    "
+        "Lisp mode\n"));
+  mch_msg(_("  -A                    "
+        "Arabic mode\n"));
+  mch_msg(_("  -F                    "
+        "Farsi mode\n"));
+  mch_msg(_("  -H                    "
+        "Hebrew mode\n"));
+  mch_msg(_("  -V[N][file]           "
+        "Be verbose [level N][log messages to file]\n"));
+  mch_msg(_("  -D                    "
+        "Debugging mode\n"));
+  mch_msg(_("  -n                    "
+        "No swap file, use memory only\n"));
+  mch_msg(_("  -r, -L                "
+        "List swap files and exit\n"));
+  mch_msg(_("  -r <file>             "
+        "Recover crashed session\n"));
+  mch_msg(_("  -u <vimrc>            "
+        "Use <vimrc> instead of the default\n"));
+  mch_msg(_("  -i <shada>            "
+        "Use <shada> instead of the default\n"));
+  mch_msg(_("  --noplugin            "
+        "Don't load plugin scripts\n"));
+  mch_msg(_("  -o[N]                 "
+        "Open N windows (default: one for each file)\n"));
+  mch_msg(_("  -O[N]                 "
+        "Like -o but split vertically\n"));
+  mch_msg(_("  -p[N]                 "
+        "Open N tab pages (default: one for each file)\n"));
+  mch_msg(_("  +                     "
+        "Start at end of file\n"));
+  mch_msg(_("  +<linenum>            "
+        "Start at line <linenum>\n"));
+  mch_msg(_("  +/<pattern>           "
+        "Start at first occurrence of <pattern>\n"));
+  mch_msg(_("  --cmd <command>       "
+        "Execute <command> before loading any vimrc\n"));
+  mch_msg(_("  -c <command>          "
+        "Execute <command> after loading the first file\n"));
+  mch_msg(_("  -S <session>          "
+        "Source <session> after loading the first file\n"));
+  mch_msg(_("  -s <scriptin>         "
+        "Read Normal mode commands from <scriptin>\n"));
+  mch_msg(_("  -w <scriptout>        "
+        "Append all typed characters to <scriptout>\n"));
+  mch_msg(_("  -W <scriptout>        "
+        "Write all typed characters to <scriptout>\n"));
+  mch_msg(_("  --startuptime <file>  "
+        "Write startup timing messages to <file>\n"));
+  mch_msg(_("  --api-info            "
+        "Dump API metadata serialized to msgpack and exit\n"));
+  mch_msg(_("  --embed               "
+        "Use stdin/stdout as a msgpack-rpc channel\n"));
```

Comment:
According to the style guide this all should be aligned with the `"` on the previous line (after the relevant `(`).